level.go

```diff
@@ -0,0 +1,94 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package auto
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+)
+
+// Level defines the log level which instrumentation uses.
+type LogLevel string
+
+const (
+	// LevelUndefined is an unset log level, it should not be used.
+	LevelUndefined LogLevel = ""
```

Comment:
Having this unexported will help prevent users from accidentally using it. It is only added to easily test unset or undefined logging levels.

```suggestion
	// levelUndefined is an unset log level, it should not be used.
	levelUndefined LogLevel = ""
```

Reply:
Changed